docs/datamodel/understanding-datamodels.rst

```diff
@@ -67,8 +67,21 @@ Building a data model involves the following tasks:
 - Choose a persistent identifier scheme.
 - Configure the REST API and UI.
 
+Internal vs External representation:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The above mentioned tasks can be divided into two: Internal and external representation. 
```

Comment:
Suggestion for rephrasing in this part:

> The above mentioned tasks can be divided into two: Internal and external representation. 
￼The tasks listed above can be divided in two categories, ones that affect the internal representation of records and ones that affect the external representation.